core-java/src/test/java/com/baeldung/system/DateTimeServiceTest.java

```diff
@@ -0,0 +1,15 @@
+package com.baeldung.system;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DateTimeServiceTest {
+
+    @Test
+    public void givenClass_whenCalledMethods_thenNotNullInResult() {
+        DateTimeService dateTimeService = new DateTimeService();
+
+        Assert.assertNotNull(dateTimeService.nowPlusOneHour());
```

Comment:
@pivovarit same kind of question here, if you have any thoughts about making these more verifiable (they rely on `System.currentTimeMillis()`)